certified-connectors/CRMBot/readme.md

```diff
@@ -0,0 +1,12 @@
+Create powerful conversation experience for your customers right from your CRM system. Give users new ways to interact with your company by building engaging voice and text-based conversational interfaces, such as voice apps and chatbots, powered by AI. Connect with users on your website, mobile app, the Google Assistant, Amazon Alexa, Facebook Messenger, and other popular platforms and devices. And do it directly from your existing CRM implementation. 
```

Comment:
Could this section be given a title? Even just the name of the connector is fine. Thanks in advance!

Reply:
Should be resolved now. Thank you! @dougelliott